commons/src/main/kotlin/org/fossify/commons/views/Breadcrumbs.kt

```diff
@@ -146,86 +106,52 @@ class Breadcrumbs(context: Context, attrs: AttributeSet) : HorizontalScrollView(
         val tempPath = context.humanizePath(fullPath)
 
         itemsLayout.removeAllViews()
-        val dirs = tempPath.split("/").dropLastWhile(String::isEmpty)
-        for (i in dirs.indices) {
-            val dir = dirs[i]
-            if (i > 0) {
-                currPath += dir + "/"
+        tempPath.split("/")
+            .dropLastWhile(String::isEmpty)
+            .forEachIndexed { i, dir ->
+                if (i > 0) {
+                    currPath += "$dir/"
+                }
+                if (dir.isEmpty()) {
+                    return@forEachIndexed
+                }
+                currPath = "${currPath.trimEnd('/')}/"
+                val item = FileDirItem(currPath, dir, true, 0, 0, 0)
+                addBreadcrumb(item, i, i > 0)
```

Comment:
Everything except the first item is being activated/colored because `isLast = i > 0`.

<img src="https://github.com/user-attachments/assets/08f50912-1008-4adc-ad49-9e9842fd4400" width=264 />